app/inspect/index.tsx

```diff
@@ -33,10 +39,7 @@ const Index = () => {
           PERMISSIONS.ANDROID.RECORD_AUDIO,
           PERMISSIONS.IOS.MICROPHONE,
         ])
-        const storageStatus = await checkMultiple([
```

Comment:
Could you explain why the storage permissions check was removed? Since we need to store data locally, how are we handling storage permissions now to ensure the app can save media correctly?

Reply:
now we are storing in internal app storage only 

that permission was for external storage access.
there is no need for permission in internal app storage .